### ISSUE TITLE
PIP mode persists after video closed.

### ISSUE THREAD

**Issue by swarnenduu** [libre-tube/LibreTube#2039]
### Steps to reproduce

Issue in Debug build after #2023 issue fixed by #2031 PR

1. Run a video
2. Minimise app to go to PIP
3. Go back to app and close video.
4. Minimise app
5. PIP activates(no video playing)

Or even

1. Run a video
2. Close video (no need to go to PIP mode while playing video)
4. Minimise app
5. PIP activates(no video playing)

### Expected behavior

No PIP when no video is played.

### Actual behavior

Empty screen PIP activates.

![Screenshot_20221125-153455](https://user-images.githubusercontent.com/86847643/203957591-454043ba-a70c-4268-b89b-b95cfa8d310e.png)

Even if empty PIP is closed > then app opened > minimise app > PIP activates.

### LibreTube version

Libretube ffd7968 // Alpha

### Android version

Android 12

### Other details

Close the entire app and then open again to stop this issue. But if you play a video again, the issue starts.

The issue is not present in LibreTube beta build v0.8.0.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
